ebuild-writing/functions/src_prepare/eapply/text.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<guide self="ebuild-writing/functions/src_prepare/eapply/">
+<chapter>
+<title>Patching with eapply</title>
+<body>
+
+<p>
+The canonical way of applying patches in ebuilds is to use the package
+manager's <c>eapply</c> function.
+</p>
+
+<p>
+<c>eapply</c> was added in EAPI=6. It differs from the previously available
+<c>epatch</c> in several ways:
```

Comment:
While this is kinda useful, it feels a bit backwards to start by telling the differences. I think we should focus on what the function does (as an information for new users), and perhaps leave differences to the old function in a separate section at the end.